lib/manager/nuget/extract.ts

```diff
@@ -112,6 +113,42 @@ export async function extractPackageFile(
     return { deps };
   }
 
+  if (packageFile.endsWith('global.json')) {
+    const deps: PackageDependency[] = [];
+    let manifest: MsbuildGlobalManifest;
+
+    try {
+      manifest = JSON.parse(content);
+    } catch (err) {
+      logger.debug({ fileName: packageFile }, 'Invalid JSON');
+      return null;
+    }
+
+    if (manifest['msbuild-sdks'] === undefined) {
+      logger.debug(
+        { fileName: packageFile },
+        'This global.json is not a Nuget file'
+      );
+      return null;
+    }
+
+    for (const depName of Object.keys(manifest['msbuild-sdks'])) {
+      const sdk = manifest['msbuild-sdks'][depName];
+      const currentValue = sdk[0];
+      const dep: PackageDependency = {
+        depType: 'nuget',
+        depName,
+        currentValue,
+        datasource: null,
+        skipReason: SkipReason.UnsupportedDatasource,
```

Comment:
```suggestion
```

This disables all deps and isn't what i suggested.

Reply:
@viceice I tried to fix this now, but to be honest it's a bit hard for me to understand such a short text provided by you. I found only the `SkipReason`. There is no `skip-reason`. I searched with `grep -rnw '.' -e 'skip-reason'`, and no luck. If I use `SkipReason`, it has the format of `UnsupportedDatasource`, not "unsupported datasource". I really apologize if I made a mistake again this time. Could you please guide me in a more verbose form if possible?